.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Also tell us, what did you expect to happen? If applicable, add screenshots to help explain your problem.
+      description: Tell us what you see!
+      placeholder: 
```

it'd be nice to have a placeholder here
```suggestion
      placeholder: I tried to install https://store.gx.me/mods/dd7dj5/lofi-chill/, and midway through the installation it froze
```